app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
```

Comment:
Consider renaming `parts` to `command_parts` for better clarity and readability, as suggested in the previous review.